Sources/RNLibreToolsiOS13/LibreToolsResources/Sensor.swift

```diff
@@ -482,17 +482,43 @@ class Sensor: ObservableObject {
         return crcReport.contains("FAILED") && history.count > 0
     }
 
-    func detailFRAM() throws {
+    struct sensorInfo {
+        var type: SensorType
+        var family: SensorFamily
+        var region: SensorRegion
+        var serial: String
+        var state: String
+        var lastReadingDate: Date
+        var age: Int
+        var maxLife: Int
+        var initializations: Int
+    }
+
+    func detailFRAM() -> AnyObject {
```

Comment:
why not just `sensorInfo` instead of AnyObject?

Also should be `open func detailFRAM()` and you need to implement override in LibrePro.swift